clouddq/classes/dataplex_entity_schema.py

```diff
@@ -38,21 +42,43 @@ def from_dict(
             error_msg=f"Entity {entity_id}: must define non-empty value: 'fields'.",
         )
 
+        if kwargs.get("partitionFields"):
+            partition_fields = kwargs.get("partitionFields")
```

Comment:
where are you parsing this into DataplexEntityPartitionSchemaField?

Reply:
parsed it to DataplexEntityPartitionSchemaField class